### ISSUE TITLE
Add schema for storing issues with applications

### ISSUE THREAD

**Issue by aroman** [elementary/houston#84]
I'm thinking the way to do this is add an `issues` array field to `ApplicationSchema`, where each `issue` is its own `IssueSchema` which looks like this:

```js
// IssueSchema
{
    githubId: 123, // e.g. github.com/user/app/issues/123
    body, // the actual content of the issue, e.g. automated output
    title, // the title of the issue, e.g. "Missing .desktop file"
}

**Comment by fabianthoma** [comment]
You mean so we can track the issues on github and get the status each time the user visits the dashboard?
If that'S the reason, sure I'm fine with that solution

**Comment by fabianthoma** [comment]
Fixed in #88
